src/PerfViewExtensions/GlobalSrc/Global.csproj

```diff
@@ -11,24 +11,6 @@
     <AssemblyName>Global</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <!-- Determine if we are building 'next to' the main PerfView sources -->
```

Comment:
Can you clarify the purpose of the extension mechanism? Is this something just used for demos, or is it / has it been used in the past to provide core functionality?